clients/drcachesim/common/trace_entry.h

```diff
@@ -191,6 +191,32 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CPU_ID,
 
+    /**
+     * The marker value contains the function id defined by user in droption
+     */
+    TRACE_MARKER_TYPE_FUNC_ID,
+
+    /**
+     * The marker value contains the return address of the function call,
+     * whose id is specified by the closest previous FUNC_ID marker entry
+     * XXX: replace return address with callstack information in the future
```

Comment:
This is a comment to us, not the user, so put it in a separate regular (non-doxygen) comment (add the i# too)

Reply:
Done